File: lib/antlr-2.7.5/examples/cpp/IDL/Main.cpp
#include <iostream>

#include "IDLLexer.hpp"
#include "IDLParser.hpp"

int main()
{
	ANTLR_USING_NAMESPACE(std)
	ANTLR_USING_NAMESPACE(antlr)

	try {
		IDLLexer lexer(cin);
		IDLParser parser(lexer);
		parser.specification();
	}
	catch( ANTLRException& e )
	{
		cerr << "exception: " << e.getMessage() << endl;
		return -1;
	}
	catch(exception& e)
	{
		cerr << "exception: " << e.what() << endl;
		return -1;
	}
	return 0;
}

File: lib/antlr-2.7.5/examples/csharp/IDL/Main.cs
using System;
using antlr;

public class AppMain {
  public static void Main(string[] args) {
    try {
      IDLLexer lexer = new IDLLexer(new ByteBuffer(Console.OpenStandardInput()));
      IDLParser parser = new IDLParser(lexer);
      parser.specification();
    } catch(Exception e) {
      Console.Error.WriteLine("exception: "+e);
    }
  }
}

